packages/server/api.js

```diff
@@ -606,10 +610,11 @@ module.exports = async function attachAPI(app, {wss, db, dbDir}) {
     }
   ])
 
-  app.post('/api/delete-channel', [
-    ...middleware.loadVarFromBody('channelID'),
+  app.delete('/api/channels/:channelID', [
+    ...middleware.loadSessionID('sessionID'),
     ...middleware.getSessionUserFromID('sessionID', 'sessionUser'),
     ...middleware.requireBeAdmin('sessionUser'),
+    ...middleware.loadVarFromParams('channelID'),
```

Comment:
Hey, should we just use `loadVar` which tries request.body and falls back to query params?

Reply:
The spec doesn't specifically allow or disallow request bodies for DELETE requests, for example.

Reply:
Okay, that sounds reasonable.